src/gedi_subset/gedi_utils.py

```diff
@@ -198,8 +216,16 @@ def subset_hdf5(
         HDF5 group to subset (typically an ``h5py.File`` instance).
     aoi : gpd.GeoDataFrame
         Area of Interest.  The subset is limited to data points that fall within this
-        area of interest, as determined by the `lat_lowestmode` and `lon_lowestmode`
-        datasets of each `"BEAM*"` group within the HDF5 file.
+        area of interest, as determined by the latitude and longitude datasets of each
+        `"BEAM*"` group within the HDF5 file.
+    lat: str
+        Name of the latitude dataset used for the resulting ``GeoDataFrame`` geometry.
+    lon: str
+        Name of the longitude dataset used for the resulting ``GeoDataFrame`` geometry.
+    beam_filter: Callable[[h5py.Group], bool] = fp.always(True)
+        Call to the beam_filter function located in subset.py. The function returns a
+        boolean value that dictates whether the `BEAM` group falls within the list of
+        beams to subset.
```

Comment:
This does _not_ call the `beam_filter` function in `subset.py`

```suggestion
    beam_filter: Callable[[h5py.Group], bool] = fp.always(True)
        Callable used to determine whether or not a top-level BEAM subgroup within the
        specified ``hdf5`` group should be included in the subset. This callable is
        called once for each subgroup that has a name prefixed with `"BEAM"`. If not
        supplied, the default callable always returns ``True``, such that every
        ``"BEAM*"`` subgroup is included. For convenience, the predicate functions
        py:`is_coverage_beam` and py:`is_power_beam` may be used. Further, the function
        returned by calling py:`beam_filter_from_names` with a specific list of BEAM
        names may be used.
```